app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -213,7 +216,36 @@ def process_template(object)
 
     # Add the inventory object for the hardware:
     hw_object = hw_collection.find_or_build(template_object)
-    hw_object.memory_mb = ManageIQ::Providers::Kubevirt::MemoryCalculator.convert(domain.memory, 'Mi')
+    hw_object.memory_mb = ManageIQ::Providers::Kubevirt::MemoryCalculator.convert(domain.resources.requests.memory, 'Mi')
+    hw_object.cpu_cores_per_socket = domain.cpu.cores
+    hw_object.cpu_total_cores = domain.cpu.cores
+    hw_object.guest_os = metadata.labels.os
+
+    # Add the inventory objects for the disk:
+    domain.devices.disks.each do |disk|
+      disk_object = disk_collection.find_or_build_by(
+        :hardware    => hw_object,
+        :device_name => disk.name
+      )
+      disk_object.device_name = disk.name
+      disk_object.location = disk.volumeName
+      disk_object.device_type = 'disk'
+      disk_object.present = true
+      disk_object.mode = 'persistent'
+      disk_object.controller_type = disk.disk.dev
+      # TODO what do we need more? We are missing reference to PV or PVC
```

Comment:
@agrare How can we build a relationship between disk and persistentvolume or/and persistentvolumeclaim?

Reply:
A disk has a polymorphic backing that you can use, backing_id, backing_type

Reply:
@Ladias is there anything special that we need to do in graph refresh to use it as @agrare suggested?